README.md

```diff
@@ -13,6 +13,15 @@ Tree-sitter is a parser generator tool and an incremental parsing library. It ca
 - **Robust** enough to provide useful results even in the presence of syntax errors
 - **Dependency-free** so that the runtime library (which is written in pure C) can be embedded in any application
 
+## Editors
+
+The following editors are making use of tree-sitter for syntax highlighting and other features:
+
+- [Neovim](https://github.com/neovim/neovim): a Vim-fork focused on extensibility and usability.
+- [helix](https://github.com/helix-editor/helix): a post-modern modal text editor.
+- [emacs-tree-sitter](https://github.com/emacs-tree-sitter/elisp-tree-sitter): bindings for Emacs 29+.
```

Comment:
Should probably remove "29+", see their main page:

> For Emacs 29+, please use [the built-in integration](https://lists.gnu.org/archive/html/emacs-devel/2022-11/msg01443.html) instead of this package.

Emacs 29+ has a built-in module into Emacs, but not the same to [emacs-tree-sitter]()/[elisp-tree-sitter]().